pkg/plugins/golang/v3/webhook.go

```diff
@@ -103,9 +103,11 @@ func (p *createWebhookSubcommand) InjectResource(res *resource.Resource) error {
 	}
 
 	// check if resource exist to create webhook
-	if r, err := p.config.GetResource(p.resource.GVK); err != nil {
+	loadedResource, err := p.config.GetResource(p.resource.GVK)
+	if err != nil || !loadedResource.HasAPI() {
 		return fmt.Errorf("%s create webhook requires a previously created API ", p.commandName)
-	} else if r.Webhooks != nil && !r.Webhooks.IsEmpty() && !p.force {
+	}
+	if loadedResource.Webhooks != nil && !loadedResource.Webhooks.IsEmpty() && !p.force {
 		return fmt.Errorf("webhook resource already exists")
 	}
```

Comment:
After rereading it again I found that the check was already there but it had a bug: if the `err != nil` that doesn't guarantee the API is created. We need to check if the loaded resource `HasAPI()` as well.

So it now all works like this:

- `Path` is always set for core resources. `create api` fails if the `Path` starts with `k8s.io`
- `create webhook` fails if there is no scaffolded API

@Adirio @estroz PTALA